WISE-Community/WISE
# Upgraded to angular 9.1

This change upgrades Angular from 8 to 9.1 and by default uses ivy and AoT compiler for development.

**To test**
Run ```npm install``` to get the updates before testing. If you have compile issues, try completely deleting node_modules folder and running ```npm install```

Test both dev (```npm run build-dev```) and prod (```npm run build-prod```) builds and make sure that everything works as before.

@breity somehow I broke the tooltip background (in run settings dialog for creating and editing a run), and it no longer shows up. I couldn't fix it. According to an article, AngularMaterial was also updated in 9.1 (https://blog.angular.io/version-9-of-angular-now-available-project-ivy-has-arrived-23c97b63cfa3), which might have caused this change. Can you please take a look?
<img width="238" alt="Screen Shot 2020-06-04 at 8 30 18 AM" src="https://user-images.githubusercontent.com/119416/83777180-fa40fb80-a63d-11ea-85ed-dd1d33082c66.png">

Resolves #2305
